apps/storybook/src/SearchBox.stories.tsx

```diff
@@ -0,0 +1,130 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import {
+  SearchBox,
+  SearchBoxProps,
+  IconButton,
+  Text,
+  Icon,
+  VerticalDivider,
+} from '@itwin/itwinui-react';
+import {
+  SvgAirplane,
+  SvgCaretDownSmall,
+  SvgCaretUpSmall,
+  SvgCloud,
+} from '@itwin/itwinui-icons-react';
+
+export default {
+  component: SearchBox,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Input/SearchBox',
+} as Meta<SearchBoxProps>;
+
+export const Basic: Story<SearchBoxProps> = (args) => {
+  return (
+    <SearchBox expandable {...args}>
+      <input type='search' placeholder='Search...' />
+    </SearchBox>
+  );
+};
+
+export const Static: Story<SearchBoxProps> = (args) => {
```

Comment:
this is interesting. shouldn't "expandable" be the special option, and the static one should be the default?

Reply:
Probably, will update that.

Reply:
Added stories that should be usefull. Will update website when component is done.